mkt/extensions/tests/test_validator.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+from rest_framework.exceptions import ParseError
+
+from mkt.site.tests import TestCase
+from mkt.extensions.validation import ExtensionValidator
+
+
+class TestExtensionValidator(TestCase):
+    """
+    Tests the ExtensionValidator class. The following methods are tested in
+    the TestExtensionViewSetPost test case instead, as part of an end-to-end
+    workflow:
+
+    * ExtensionValidator.validate_file
+    * ExtensionValidator.validate_json
+    """
+    def setUp(self):
+        self.validator = ExtensionValidator()
+        super(TestExtensionValidator, self).setUp()
+
+    def test_name_missing(self):
```

Comment:
You should test `validate` as well. With a file-like object (IIRC `StringIO` or a mock should work).